securecn/resource_portshift_k8s_cluster.go

```diff
@@ -92,7 +92,10 @@ func ResourceCluster() *schema.Resource {
 			IstioAlreadyInstalledFieldName:            {Type: schema.TypeBool, Optional: true, Default: false, Description: "if false, istio will be installed, otherwise the controller will use the previously installed istio"},
 			IstioVersionFieldName:                     {Type: schema.TypeString, Optional: true, Default: nil, Computed: true, Description: "if istio already installed, this specifies its version"},
 			IstioIngressEnabledFieldName:              {Type: schema.TypeBool, Optional: true, Computed: true, Description: "If installing Istio, use Istio ingress"},
-			IstioIngressAnnotationsFieldName:          {Type: schema.TypeMap, Elem: schema.TypeString, Optional: true, Default: map[string]string{}, Description: "If enabling Istio ingress, use Istio these ingress annotation"},
+			IstioIngressAnnotationsFieldName:          {Type: schema.TypeMap,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				}, Optional: true, Description: "If enabling Istio ingress, use Istio these ingress annotation"},
```

Comment:
the text doesn't make sense. maybe "when enabling Istio ingress, use **these Istio** ingress annotations"